### ISSUE TITLE
`dumper_enabled_host?` is not defined? 🤔

### ISSUE THREAD

**Issue by yasulab** [kenn/dumper#13]
I added Dumper agent in [CoderDojo Japan](https://coderdojo.jp/) website and found that the `dumper_enabled_host?` method seems not defined.

Search - dumper_enabled_host?
https://github.com/dumperhq/dumper/search?utf8=%E2%9C%93&q=dumper_enabled_host%3F&type=

The code and error I found are here:

- Code: https://github.com/coderdojo-japan/coderdojo.jp/pull/326/files#r186308427
- Error: https://travis-ci.org/coderdojo-japan/coderdojo.jp/builds/375686170#L996

Hope this helpful ;)

**Comment by kenn** [comment]
Right, that's a pseudo code that demonstrates that you can determine on which server you want to run the agent.

**Comment by kenn** [comment]
Maybe we could just remove it from the commented line.

**Comment by yasulab** [comment]
Oh, didn't notice it...!! 🙈💦

Yes, maybe removing or making it easier to notice is better. 😆✨
